src/index.js

```diff
@@ -82,6 +82,7 @@ const updateGithubRepoDependency = decorateFunctionLogger(({
             gitlabToken,
             logger
         })
+        .catch(() => '')
```

Comment:
@pwmckenna Not sure what to put here. Maybe just the module name and its version?

Reply:
seems fine. we can update if we want more